app/views/events/show.html.erb

```diff
@@ -35,16 +35,6 @@
     <%= @event.place.full_adress %>
   </p>
 
-<<<<<<< HEAD
-  <% if can? :destroy, @event %>
```

Comment:
I think to have possibility to remove event, you need to have this three lines for destroy.